tests/passes/router/test_shortest_path_router.py

```diff
@@ -36,6 +36,17 @@ def router3() -> ShortestPathRouter:
     return ShortestPathRouter(connectivity)
 
 
+@pytest.fixture(name="router4")
+def router_fixture4() -> ShortestPathRouter:
```

Comment:
No need to specify the name of the fixture (the name of the definition does the job).

```suggestion
@pytest.fixture
def router4() -> ShortestPathRouter:
```